docs/versioned_docs/version-2.11.0/deprecated/ugcstor.md

```diff
@@ -0,0 +1,2496 @@
+---
+id: ugcstor
+title: SPC based cStor Guide
+---
+
+:::note Deprecation Notice
+   cStor `out-of-tree external` provisioners will be deprecated by Dec 2021 in favor of the corresponding CSI Drivers. The out of tree provisioners for cStor will stop working from Kubernetes 1.22 and forward as the version of the custom resources used by those provisioners will be deprecated. We strongly recommend you plan for migrating your volumes to [cStor CSI](https://github.com/openebs/upgrade/blob/master/docs/migration.md) as early as possible.  
+
+   For detailed instructions on how to get started with new cStor Operators please refer [here](/docs/next/ugcstor-csi.html).
+
+   If you have any questions or need help with the migration please reach out to us on our Kubernetes Community slack channel [#openebs](https://kubernetes.slack.com/archives/CUAKPFU78)
+:::
+
+[![OpenEBS configuration flow](../assets/3-config-sequence.svg)](../assets/3-config-sequence.svg)
+
+This user guide section provides the operations need to performed by the User and the Admin for configuring cStor related tasks. 
+
+## User operations
+
+[Provisioning a cStor volume](#provisioning-a-cStor-volume)
+
+[Monitoring a cStor Volume](#monitoring-a-cStor-Volume)
+
+[Backup and Restore](#backup-and-restore)
+
+[Snapshot and Clone of a cStor Volume](#snapshot-and-clone-of-a-cStor-volume)
+
+[Upgrading the software version of a cStor volume](#Upgrading-the-software-version-of-a-cStor-volume)
+
+[Provisioning sample application with cStor](#Provisioning-sample-application-with-cstor)
+
+[Deleting a cStor Volume](#deleting-a-cStor-volume)
+
+[Patching pool deployment by adding or modifying resource limit and requests](#patching-pool-deployment-resource-limit)
+
+## Admin operations
+
+[Creating cStor storage pools](#creating-cStor-storage-pools)
+
+[Setting Pool Policies](#setting-pool-policies)
+
+[Creating cStor storage classes](#creating-cStor-storage-class)
+
+[Setting Storage Polices](#cstor-storage-policies)
+
+[Monitoring a cStor Pool](#monitor-pool)
+
+[Setting Performance Tunings](#setting-performance-tunings)
+
+[Upgrade the software version of a cStor pool](#Upgrade-the-software-version-of-a-cStor-pool)
+
+[Expanding cStor pool to a new node](#expanding-cStor-pool-to-a-new-node)
+
+[Expanding size of a cStor pool instance on a node by expanding the size of cloud disks](#expanding-size-of-a-cStor-pool-instance-on-a-node-add)
+
+[Expanding size of a cStor pool instance on a node by add physical/virtual disks to a pool instance](#expanding-size-of-a-cStor-pool-instance-on-a-node-add-disk)
+
+[Expanding the cStor Volume Capacity](#expanding-size-of-a-cStor-volume)
+
+[Scaling up cStor Volume Replica](#scaling-up-of-cvr)
+
+[Scaling down cStor Volume Replica](#scaling-down-of-cstor-volume-replica)
+
+## User Operations
+
+### Provisioning a cStor volume
```

Comment:
> line no: 20 -  Provisioning a cStor volume](#provisioning-a-cStor-volume)

This heading requires custom id, as you can see in the above hyperlink. The default one will provide id but all the letters will be in lowercase, so in this case you have to specify the id 

```suggestion
### Provisioning a cStor volume {#provisioning-a-cStor-volume}
```

Reply:
look for other places, if the link contains the text same as heading (same textcase also, no special chars) it will be fine, else you have to provide custom id for the heading same as in the link

Reply:
Actually Id should be `lowercased`

Reply:
For eg. `#monitoring-a-cStor-Volume` is not a fully lowercased id. ( for it, you will have to add custom id )